linux/installer/common/gen_source/gen_source.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
It is not a good solution to hard code the python version.   It may be better to change the style to make it compatible.